obal/data/roles/build_package/tasks/koji.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: "Confirm package is whitelisted"
+  package_whitelist_check:
+    releasers_conf: "{{ inventory_dir }}/rel-eng/releasers.conf"
+    spec_file_path: "{{ spec_file.files[0].path }}"
```

Comment:
I guess this can be `spec_file_path` now?

https://github.com/theforeman/obal/blob/f2dcda2314eb8e84b1cae69c65b201df192c1248/obal/data/roles/spec_file/tasks/main.yml#L10

Reply:
You are correct!